src/Viewer.tsx

```diff
@@ -397,6 +398,8 @@ function Viewer(): ReactElement {
         return;
       }
 
+      loadInitialViewerStateFromParams(useViewerStateStore, searchParams);
```

Comment:
Initial viewer state for non-dependent params is done before dataset starts to load in